Directory.Build.props

```diff
@@ -104,6 +104,23 @@
     <PublicApiAnalyzersVersion>3.3.4</PublicApiAnalyzersVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net10.0'">
+    <NetNineRuntimeVersion>10.0.0</NetNineRuntimeVersion>
```

Comment:
these need to be on the respective preview versions from the link to the .NET 10 download page